dbms/src/WindowFunctions/tests/gtest_last_value.cpp

```diff
@@ -326,7 +326,7 @@ class LastValue : public DB::tests::WindowTest
             {
                 mock_frame.end = buildRangeFrameBound(
                     tipb::WindowBoundType::Following,
-                    tipb::RangeCmpDataType::Int,
```

Comment:
Why modify existing test cases instead of adding new ones?

Reply:
> Why modify existing test cases instead of adding new ones?

Because the calculation of time and int are the same, we only need to ensure that tipb::RangeCmpDataType::DateTime could enter the correct code path.

Reply:
This can be confusing.
I prefer to 
```
auto types{int, time}
for (type : types)
{
    xxx
}
```

Reply:
> This can be confusing. I prefer to
> 
> ```
> auto types{int, time}
> for (type : types)
> {
>     xxx
> }
> ```

okk